exp-models/addon/models/account.js

```diff
@@ -2,14 +2,34 @@
 Manage data about one or more documents in the accounts collection
  */
 
+import moment from 'moment';
+
+import Ember from 'ember';
 import DS from 'ember-data';
 import JamModel from '../mixins/jam-model';
 
+var Profile = Ember.Object.extend({
+    birthday: null,
+    age: Ember.computed('birthday', function() {
+        var bd = moment(this.get('birthday'));
+        return moment(new Date()).diff(bd, 'days');
+    })
+});
+
 export default DS.Model.extend(JamModel, {
     username: DS.attr('string'),
     password: DS.attr('string'),
-    profiles: DS.attr(),  // TODO: Pages using profiles will be fragile unless there is a clear, explicit contract of what fields ember can always expect to be present. This nested document doesn't inherently specify as such.
-
+    _profiles: DS.attr(), // TODO: Pages using profiles will be fragile unless there is a clear, explicit contract of what fields ember can always expect to be present. This nested document doesn't inherently specify as such.
```

Comment:
I understand the desire to have an ember object here, but aliasing the field can get very... messy.

It would be cleaner to name the computed observable field something completely different, like `profileData`. That would let us eliminate the custom serializer logic, and avoid any confusion when someone tries to use `createRecord` based on the layout of the JSON document.

(When individual fields break the rules on a case-by-case basis, it can get very confusing, especially since otherwise our models are explictly designed to look like the server fields wherever possible)